packages/thumbprint-react/components/ModalCurtain/test.tsx

```diff
@@ -47,23 +47,23 @@ describe('ModalCurtain', () => {
 
         mount(
             <ModalCurtain onCloseClick={jest.fn}>
-                {args => {
+                {(args): JSX.Element => {
                     obj = args;
```

Comment:
What's the change here? I don't think I understand it.

Reply:
not sure what you're asking about specifically here, i just changed it to make the type checking pass. the `children` prop of `ModalCurtain` is a function that returns `JSX.Element`, but `null` is not included in that, so I switched it to an empty `div`